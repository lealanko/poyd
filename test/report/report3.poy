(* Build and Report cross_references: names:()) *)
read ("1.fas")
build (1)
report (cross_references: names:())