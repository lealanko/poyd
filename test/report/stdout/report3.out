File References:
                  
  Terminal
  t1
  t2
  t3
  t4
  t5
  t6
  t7
  t8
  t9
  t10
  t11
  t12
  t13
  t14
  t15
  t16
  t17
  t18
  t19
  t20
  t21
  t22
  t23
  t24
  t25
  t26
  t27
  t28
  t29
  t30
  t31
  t32
  t33
  t34
  t35
  t36
  t37
  t38
  t39
  t40
  t41
  t42
  t43
  t44
  t45
  t46
  t47
  t48
  t49
